style.css

```diff
@@ -55,23 +146,185 @@ ul {
   color: #172b4d;
 }
 
+.tertiary_color {
+  color: #3a4a42;
+}
+
+/* Buttons */
+
+.btn1 {
+  background-color: #36b37e;
+  height: 48px;
+  width: 122px;
+  border-radius: 4px;
+  padding: 12px;
+  border: none;
+  cursor: pointer;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.btn1:enabled {
+  background-color: #36b37e;
+}
+
+.btn1:hover {
+  background-color: #36b37f;
+}
+
+.btn1:active {
+  background-color: #008552;
+}
+
+.btn_resume {
+  background-color: #36b37e;
+  width: 159px;
+}
+
+/* Classes */
+
+.article_about_me p {
+  margin-bottom: 35px;
+  border-radius: 8px;
+  height: 474px;
+  width: 327px;
+}
+
+.article_card {
+  border-radius: 8px;
+  height: 352px;
+  width: 327px;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly give between your cards the appropriate spacing as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206039027-d0611732-430d-4aba-bd3c-2a59c37b31d5.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206039123-6fb28187-5de9-4515-ac64-267b3c47b7c1.png)